### PR TITLE
all wiki content added

### PR DESCRIPTION
**Pull request by akshit13**

### CODE REVIEW

File: _community/blogposts-reports.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Blogposts/Reports
+permalink: /community/blogposts-reports
```
Comment (lavish205) [inline _community/blogposts-reports.md:4]: add trailing slash

File: LICENSE.txt [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
```
Comment (lavish205) [inline LICENSE.txt:21]: why licence file was deleted?

File: _community/events.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Events
+permalink: /community/events
```
Comment (lavish205) [inline _community/events.md:4]: trailing slash

File: _community/scenes-projects.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Scenes/Projects
+permalink: /community/scenes-projects
```
Comment (lavish205) [inline _community/scenes-projects.md:4]: trailing slash

File: _community/scenes-projects.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    * Akshil Shah : [Image cube](http://akshil.in/vr/) & [3D Letter](http://akshil.in/vr/akshil.html).
+    * Heet Shah : [Image gallery](http://heet.heetshah.com/) 
+    * Ronil Patel : [Drums & Balls](https://devilsown111.github.io/vr/ronilpatel.html).
+    * Abhinav Baldha : [3D Hello](https://abhinavbaldha.github.io/webVR1/vr5.html).
+    * Tarvik Sangani : [Hello VR](https://tarviksangani.github.io/VR/hello_VR.html).
+    * Bansi Dongha : [Amazing animated 3D model](https://bansi511.github.io/firstweb/vr.html).
+* Q2 2017 :
+  * Robin created [VR images gallery](https://robinkoshy.github.io/MozVR/).
+  * Scenes created during Hack Night event:
+    * @vibhanshuchaudhary - https://vibhanshuchaudhary.github.io/HackNightIndia/hacknight.html
```
Comment (lavish205) [inline _community/scenes-projects.md:46]: we can create link for their twitter handle

File: _community/superstarts.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Contributors aka WebVR-India-Superstarts ;)
+permalink: /community/superstars
```
Comment (lavish205) [inline _community/superstarts.md:4]: trailing slash

**Review by lavish205 — COMMENTED** [review]
@akshit13 can you please go through them once again?

File: LICENSE.txt (hunk 2 above)
Comment (akshit13) [reply LICENSE.txt:21]: @lavish205 because we had multiple licence file in the repo. I deleted one of them.

**Pull request merged**